Makefile

```diff
@@ -42,6 +42,7 @@ quicktest:
 	docker-compose run --rm test bash -c "vendor/bin/behat --stop-on-failure --strict --append-snippets"
 
 test: appfortests
+	sleep 3 # let other containers get settled
```

Comment:
Can you use `whenavail` or at least include in your comment which container(s) are the problem?